features/hyphenate-limit-chars.yml

```
@@ -0,0 +1,7 @@
+name: Hyphenate limit chars
+description: The `hyphenate-limit-chars` CSS property sets the number of characters in a word before it is hyphenated and the minimum number of characters before and after the hyphen.
```

```suggestion
description: The `hyphenate-limit-chars` CSS property sets the number of characters a word must have to be hyphenated and the minimum number of characters before and after the hyphen.
```
Slight reword to avoid using `before` with 2 different definitions.

Alternatively- 

```suggestion
description: The `hyphenate-limit-chars` CSS property sets the number of characters in a word before it is hyphenated and the minimum number of characters on either side of the hyphen.
```